src/compiler/commandLineParser.ts

```diff
@@ -437,7 +437,6 @@ export const commonOptionsWithBuild: CommandLineOption[] = [
         name: "generateTrace",
         type: "string",
         isFilePath: true,
-        isCommandLineOnly: true,

```

Comment:
Please revert all changes except this since we still don’t want the option to be in api but allowed ok to be in tsconfig

Reply:
no problem, done.